### PR TITLE
feat(like): allow regular expression in like operator

### PR DESCRIPTION
**Pull request by moasl**
Allow regular expression in like operator

### CODE REVIEW

File: test/unit/parser.test.js [hunk 1]
```diff
@@ -1346,6 +1346,18 @@ describe('Parser', () => {
...
     });
+
+    it('should match true on isRegex', () => {
+      const parser = new Parser('/corge like "[a-z]{1}"');
+      const filter = parser.parse().value;
+      assert.isTrue(filter.match(src, true));
+    });
+
+    it('should match false on isRegex', () => {
+      const parser = new Parser('/foo like "\\w"');
```
Comment (greuze) [inline test/unit/parser.test.js]: Check that this test should really return `false`, as it seems to be a valid regexp.

**Comment by greuze** [comment]
Brought commit from https://github.com/lyudmila-petrova/spleen-node/commit/c379c7ed1295e3c3c913771e80ee38b2f5c668fe that fixes to allow a target array (variable).

This commit decreases the coverage in `parser.js`, that will be fixed as soon as possible

**Comment by greuze** [comment]
Latest commit https://github.com/Telefonica/spleen-node/pull/1/commits/e829f60cb25dc6e963f41378e4793568edb96168 recovers 100% code coverage in tests

File: test/unit/parser.test.js (hunk 1 above)
Comment (moasl) [reply test/unit/parser.test.js]: It is because /foo is not a string. I change it

**Review by meji92 — APPROVED** [review]

**Review by marivipelaez — APPROVED** [review]

**Review by xhermosilla — APPROVED** [review]

**Pull request merged**